### ISSUE TITLE
Blocking/Sketching artifacts after sudden convergence

### ISSUE THREAD

**Issue by Anguliachao** [zjysteven/controlnet_tile#2]
Dear author,
     thank you for the detailed script as tutorial, I'm also trying to train a tile version controlnet as learning purpose, while I find it difficult to replicate the official tile-Controlnet results, I manged to find your repo quite useful and try to replicate as my first step. However I found your max training steps are just max capped at 10K steps and I set more max training steps . I observed sudden convergence after 3-5K steps by using 4xV100 gpu, however while the training loss remains stable, I observed the validation outputs have certain visual artifacts appeared , such as blocking, downsampling artifacts, unexpected sketches or noise would occur after training for longer than 20k  steps (below pictures attached, step 4k vs step30k), so I was confused whether this occur to you or have u ever trained a tile model with longer steps(>10K)/multi-scale fine-tuning, would you care to share some tips or kind advice? Many thanks
![3999](https://github.com/user-attachments/assets/08841620-133c-4f3c-8128-8139eb3ac1da)
![30499](https://github.com/user-attachments/assets/fa3a500d-b47b-4c08-8c42-99529aa64126)

**Comment by zjysteven** [comment]
Hi. No unfortunately I have never trained for that many steps, so I'm afraid that I can't help much on that.

**Comment by Anguliachao** [comment]
Noted, thank you for the prompt response.
